dask_expr/_groupby.py

```diff
@@ -86,6 +113,8 @@ class SingleAggregation(ApplyConcatApply):
 
     @classmethod
     def chunk(cls, df, by=None, **kwargs):
+        if hasattr(by, "dtype"):
+            by = [by]
```

Comment:
Since you highlighted this as something you didn't like earlier: we cant use `_convert_to_list` here, so I think this explicit logic is fine. (not sure what else we can do)

Reply:
Yeah same